MozziGuts.cpp

```diff
@@ -35,6 +35,9 @@
 #include <uart.h>
 #include <Ticker.h>
 #endif
+#ifdef __DAC_MCP__
```

Comment:
For consistency, please move this to MozziGuts.h (where all the other AudioConfigXYZ.h are included).